src/lib/spinel/radio_spinel_impl.hpp

```diff
@@ -418,7 +419,7 @@ otError RadioSpinel<InterfaceType, ProcessContextType>::RestoreDatasetFromNcp(vo
 {
     otError error = OT_ERROR_NONE;
 
-    otPlatSettingsInit(mInstance);
+    Instance::Get().template Get<SettingsDriver>().Init();
```

Comment:
```suggestion
    Instance::Get().Get<SettingsDriver>().Init();
```
This keyword is not required here.

Reply:
It turns out that clang still requires this keyword here. GCC doesn't. Please ignore this suggestion and revert. :sweat_smile: